Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
Thanks.  I guess I was hoping to keep it a little decoupled from Travis, in case we ever need to move to a different CI service.  Also, it would be nice if it could still be run manually outside of CI with `npm run checkCommit` (with the caveat that if your master is not up-to-date with eslint/eslint's master, it may be inaccurate).

I'll look into the GitHub API, thanks.